cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -54,7 +50,8 @@ pub(crate) fn compute_host_function_transaction_data_and_min_fee(
             &soroban_resources.footprint,
         )
         .context("cannot estimate maximum transaction size")?,
-        contract_events_size_bytes: soroban_resources.contract_events_size_bytes,
+        contract_events_size_bytes: calculate_event_size_bytes(events)
```

Comment:
Events now also include the invocation return value size (naming is a bit misleading, but that's change from before we changed any XDR and I'm not sure it's worth going through the refactoring at this point).

Reply:
You mean the size of the `SCVal` returned by the function? That's really unintuitive

Also, it would had been nice to learn about these sort of changes in advance.